lib/recorder/audio.ts

```diff
@@ -76,7 +76,10 @@ export class AudioRecorder {
    */
   private _generateObjectUrl(): void {
     // Use wav for faster and simple encoding
-    const blob = new Blob(this._audioData, { type: 'audio/wav' });
+    // Due to an iOS 16 MediaStream Recording issue found in iOS 16,
+    // We need to generate mp4 to work around this.
+    // https://bugs.webkit.org/show_bug.cgi?id=249250
+    const blob = new Blob(this._audioData, { type: 'audio/mp4' });
```

Comment:
Initially, we used `audio/wav` because most if not all of browsers support this mime type. However, now that we found that iOS16 does not support it, we cannot hard code it to use only `audio/mp4`. We should make this dynamic depending what the browser supports. I believe the [MediaRecorder](https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder/mimeType) object contains a mimeType property which returns what was chosen by the browser. Can you please test on multiple browser and see what it returns?

Reply:
If we end up using it, we should also add some tests.

Reply:
Good catch, perhaps we can make it so that it's more dynamic in nature and not just `audio/wav` or `audio/mp4` and use what MediaRecorder default selection is? Would there be any repercussions with going that route? I know in the note, `wav` was chosen since it was faster. 

I've tested it on several browser types. Not all, but here's what some of them look like. 

iOS 16 / Safari :
![image](https://user-images.githubusercontent.com/43423318/210902888-751510ec-9c95-463b-ac56-c1895867e449.png)

Chrome Desktop:
![image](https://user-images.githubusercontent.com/43423318/210902942-f8ae0b98-2ec6-491a-a689-a189f2db6cc6.png)

Firefox Desktop:
![image](https://user-images.githubusercontent.com/43423318/210902988-8527c87b-0b41-499f-ab67-2aceab66cbfd.png)

Safari Desktop:
![image](https://user-images.githubusercontent.com/43423318/210903008-7920eeb3-6df7-4ac7-9e6b-90bef8a84160.png)

Reply:
@charliesantos A quick update: I've added a unit test, but since I'm not able to run the tests as I haven't been able to actually get `npm install` working. I'd love if you could take a look at the changes and see if you can verify that the test is working! 

But, the adjusted implementation seems to be working across all of the browsers after testing it manually.